code/game/objects/items/devices/atomic_disassembler.dm

```diff
@@ -0,0 +1,156 @@
+/obj/item/device/atomic_disassembler
+	name = "P.A.D.D."
```

Comment:
If the full name is in the examine text of it I will say it is fine to remove filler attack line text.